### PR TITLE
correcting vpcId typo in the docs to vpc_id

### PR DESCRIPTION
**Pull request by aliabbasjaffri**

### CODE REVIEW

**Comment by scottslowe** [comment]
Thank you for your contribution!

However, it seems as if `vpc.vpcId` is correct here. Based on the API docs (see https://www.pulumi.com/registry/packages/awsx/api-docs/ec2/vpc/), `vpcId` is listed as an Output for `awsx.ec2.vpc`. I also tested the use of `vpc.vpcId` in a TypeScript program and didn't run into any errors.

Can you point me to an instance where you needed to use `vpc_id`?

**Comment by aliabbasjaffri** [comment]
Hello! Nopes, i was editing the section: https://www.pulumi.com/docs/guides/crosswalk/aws/vpc/#configuring-internet-and-nat-gateways-for-subnets-in-a-vpc for python. That's where i saw the discrepancy.

**Comment by scottslowe** [comment]
@aliabbasjaffri Got it; yes, I now see similar/same errors in both places.

Would you be up to extend your PR to fix not only the Python typo in both places, but also the TypeScript typo? (The TypeScript version references `vpc.id` when it should be `vpc.vpcId`.)

**Comment by aliabbasjaffri** [comment]
I can do that :-)

**Comment by aliabbasjaffri** [comment]
@scottslowe i managed to updated other occurrances of vpc id in the document for consistency.

File: themes/default/content/docs/guides/crosswalk/aws/vpc.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ import pulumi_awsx as awsx
 
 vpc = awsx.ec2.Vpc("custom")
 
-pulumi.export("vpcId", vpc.vpc_id)
+pulumi.export("vpc_id", vpc.vpc_id)
```
Comment (scottslowe) [inline themes/default/content/docs/guides/crosswalk/aws/vpc.md:192]: Sorry to be difficult, but we'd rather you left the exported Output name the same (i.e., leave it as `vpcId`) and just change the resource and property as needed. Can you revert the changes to the exports?

File: themes/default/content/docs/guides/crosswalk/aws/vpc.md [hunk 2]
```diff
@@ -336,7 +336,7 @@ import pulumi_awsx as awsx
 
 vpc = awsx.ec2.Vpc("custom", cidr_block="172.16.8.0/24")
 
-pulumi.export("vpcId", vpc.vpc_id)
+pulumi.export("vpc_id", vpc.vpc_id)
```
Comment (scottslowe) [inline themes/default/content/docs/guides/crosswalk/aws/vpc.md:339]: Same as previous comment

File: themes/default/content/docs/guides/crosswalk/aws/vpc.md [hunk 3]
```diff
@@ -426,7 +426,7 @@ import pulumi_awsx as awsx
 
 vpc = awsx.ec2.Vpc("custom", number_of_availability_zones=4)
 
-pulumi.export("vpcId", vpc.vpc_id)
+pulumi.export("vpc_id", vpc.vpc_id)
```
Comment (scottslowe) [inline themes/default/content/docs/guides/crosswalk/aws/vpc.md:429]: Same as previous comment

File: themes/default/content/docs/guides/crosswalk/aws/vpc.md [hunk 4]
```diff
@@ -643,7 +643,7 @@ import pulumi_awsx as awsx
 vpc = awsx.ec2.Vpc("custom", nat_gateways=awsx.ec2.NatGatewayConfigurationArgs(
     strategy=awsx.ec2.NatGatewayStrategy.SINGLE))
 
-pulumi.export("vpcId", vpc.vpcId)
+pulumi.export("vpc_id", vpc.vpc_id)
```
Comment (scottslowe) [inline themes/default/content/docs/guides/crosswalk/aws/vpc.md:646]: Same as previous comment

File: themes/default/content/docs/guides/crosswalk/aws/vpc.md [hunk 5]
```diff
@@ -787,7 +787,7 @@ allow_tls = aws.ec2.SecurityGroup("allowTls",
                                     cidr_blocks=["0.0.0.0/0"],
                                   )])
 
-pulumi.export("vpcId", vpc.vpc_id)
+pulumi.export("vpc_id", vpc.vpc_id)
```
Comment (scottslowe) [inline themes/default/content/docs/guides/crosswalk/aws/vpc.md:790]: Same as previous comment

File: themes/default/content/docs/guides/crosswalk/aws/vpc.md [hunk 6]
```diff
@@ -941,7 +941,7 @@ server = aws.ec2.Instance("webserver-www",
                           ami=ami.id,
                           subnet_id=vpc.public_subnet_ids.apply(lambda id: id[0]))
 
-pulumi.export("vpcId", vpc.vpc_id)
+pulumi.export("vpc_id", vpc.vpc_id)
```
Comment (scottslowe) [inline themes/default/content/docs/guides/crosswalk/aws/vpc.md:944]: Same as previous comment

**Review by scottslowe — DISMISSED** [review]
Thanks for this! I do have a request to revert the changes to the export names (we'd prefer to keep it as it was). We would prefer to only change the incorrect resource property name (i.e., the instances where Python code incorrectly references `vpc.vpcId` instead of `vpc.vpc_id` and where TypeScript code incorrectly references `vpc.id` instead of `vpc.vpcId`). Thank you!

**Comment by susanev** [comment]
@aliabbasjaffri we appreciate your contribution here and are going to merge as is, thank you!

**Review by susanev — APPROVED** [review]
